docs/source/reference/difference.rst

```diff
@@ -90,6 +90,6 @@ If these types were returned, it would be required to synchronize between GPU an
 If you want to use scalar values, cast the returned arrays explicitly.
 
   >>> type(np.sum(np.arange(3)))
-  <type 'numpy.int64'>
```

Comment:
How about using `isinstance` ?
It is accepted by Python2 and Python3.

Reply:
Used `type() ==` no to change much, but works with both 2 and 3.